velox/docs/functions/spark/math.rst

```diff
@@ -96,6 +96,14 @@ Mathematical Functions
 
     Returns Euler's number raised to the power of ``x``.
 
+.. spark:function:: expm1(x) -> double
+
+    Returns Euler's number raised to the power of ``x``, minus 1, which is "exp(x)-1" in math, while this function is more accurate than exp(num) - 1 when the num is close to zero.
```

Comment:
suggestion:
xxx is more accurate than \``std::exp(x) - 1\`` when \``x\`` is close xxx.

Nit: "exp(x)-1" -> \``exp(x) - 1\``(with some spaces added)